ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.15-periodics.yaml

```diff
@@ -723,14 +709,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.15"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-mce-disconnected
-  reporter_config:
-    slack:
-      channel: '#hypershift-kubevirt-platform'
-      job_states_to_report:
-      - failure
-      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs> :volcano:'
```

Comment:
I don't know what this sections does, but it doesn't seem that it should have been removed.

Reply:
correct me if I am wrong

Reply:
good catch. I think this is how we can get slack alerts from jobs. I didn't remove it manually, but maybe `make jobs` did it? thanks for catching it. I added it back.